docker/config-example.toml

```diff
@@ -1,26 +1,16 @@
 [general]
 
-L1_RPC_ENDPOINT = "http://l1geth:8545"
-L2_RPC_ENDPOINT = "http://l2geth:8545"
+L1_RPC_ENDPOINT = "http://l1-devnet:8545"
+L1_RPC_ENDPOINT_WEBSOCKET = "ws://l1-devnet:8546"
```

Comment:
Is this only used in other scripts?

Reply:
yes, added it here to make it consistent with the config.toml in scroll-sdk.